### PR TITLE
[Support-branch] PIM-5728, PIM-5762, PIM-5666, PIM-5738, PIM-5727, PIM-5754, PIM-5763, PIM-5622, PIM-5756

### PR DESCRIPTION
**Pull request by pierallard**
Commit fixing
- PIM-5728
- PIM-5762
- PIM-5666
- PIM-5727
- PIM-5754
- PIM-5763
- PIM-5622
- PIM-5756
- PIM-5655

All of these are GTMed on their own PR's.

### CODE REVIEW

File: features/Pim/Behat/Context/AttributeValidation.php [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+                        assertEquals('1' === $value, $attribute->isScopable());
+                        break;
+                    case 'wysiwyg_enabled':
+                        assertEquals('1' === $value, $attribute->isWysiwygEnabled());
+                        break;
+                    case 'decimals_allowed':
+                        assertEquals('1' === $value, $attribute->isDecimalsAllowed());
+                        break;
+                    case 'negative_allowed':
+                        assertEquals('1' === $value, $attribute->isDecimalsAllowed());
```
Comment (skeleton) [inline features/Pim/Behat/Context/AttributeValidation.php]: nope, should be isNegativeAllowed

File: features/filter/group.feature [hunk 2]
```diff
@@ -16,4 +16,3 @@ Feature: Filter on groups
       | filter                                                                            | result                        |
       | [{"field":"groups.code", "operator":"IN",     "value": ["substitute", "upsell"]}] | ["TSHIRT", "JACKET", "SWEAT"] |
       | [{"field":"groups.code", "operator":"NOT IN", "value": ["substitute", "upsell"]}] | ["PANT", "BOOT"]              |
-      | [{"field":"groups.code", "operator":"EMPTY",  "value": null}]                     | ["BOOT"]                      |
```
Comment (skeleton) [inline features/filter/group.feature]: why do you remove test on "empty" operator ?

File: features/Pim/Behat/Context/AttributeValidation.php [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+                        throw new \Exception(sprintf(
+                            "The attribute method '%s' is not testable, please add it in %s",
+                            $method,
+                            get_class($this)
+                        ));
+                }
+            }
+        }
+    }
+}
```
Comment (skeleton) [inline features/Pim/Behat/Context/AttributeValidation.php]: add new line at the end of file

Comment (pierallard) [inline features/Pim/Behat/Context/AttributeValidation.php]: done.

File: features/Context/FixturesContext.php [hunk 4]
```diff
@@ -1620,7 +1538,7 @@ protected function createAttribute($data)
      *
      * @return string
      */
-    protected function getAttributeType($type)
+    public function getAttributeType($type)
```
Comment (jjanvier) [inline features/Context/FixturesContext.php]: if it's used only by AttributeValidation context, then move the method

**Comment by jjanvier** [comment]
don't know if it's good to introduce even more coupling between the contexts, 
there is no other way to do that?

to be able to refresh an object without calling the getMainContext for example?

GTM otherwise

**Comment by skeleton** [comment]
GTM once fixed

File: features/filter/group.feature (hunk 2 above)
Comment (pierallard) [inline features/filter/group.feature]: Bad manip'. I reset it.

File: features/Pim/Behat/Context/AttributeValidation.php [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+use Behat\Gherkin\Node\TableNode;
+
+/**
+ * Context for the attribute validation
+ *
+ * @author    Pierre Allard <pierre.allard@akeneo.com>
+ * @copyright 2016 Akeneo SAS (http://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+class AttributeValidation extends PimContext
```
Comment (phaseinducer) [inline features/Pim/Behat/Context/AttributeValidation.php]: Please add `Context` suffix for consistency.

**Pull request merged**